src/effects/Caption.cpp

```diff
@@ -153,46 +153,28 @@ std::shared_ptr<openshot::Frame> Caption::GetFrame(std::shared_ptr<openshot::Fra
 	double fade_in_value = fade_in.GetValue(frame_number) * fps.ToDouble();
 	double fade_out_value = fade_out.GetValue(frame_number) * fps.ToDouble();
 	double right_value = right.GetValue(frame_number);
-	double background_corner_value = background_corner.GetValue(frame_number);
-	double padding_value = background_padding.GetValue(frame_number);
+	double background_corner_value = background_corner.GetValue(frame_number) * scale_factor;
+	double padding_value = background_padding.GetValue(frame_number) * scale_factor;
+	double stroke_width_value = stroke_width.GetValue(frame_number) * scale_factor;
+	double line_spacing_value = line_spacing.GetValue(frame_number);
+	double metrics_line_spacing = metrics.lineSpacing();
```

Comment:
metrics.lineSpacing() is already based on the FontSize (which has already been scaled to the preview window size) - thus, this metric is also scaled

Reply:
Makes sense! And looks like that was the original issue making spacing too large.